### DISCUSSION TITLE
Tibber integration for Cost calculation

### DISCUSSION THREAD

**Discussion by Cyrelion** [solectrus/solectrus#2988] [Q&A]
Hello,

I've successfully enabled the tibber collector in my solectrus docker installation, but how can I configure the web view to use this for the pricing calculations? I need to set a value to the electricity price entry and can't set it to a fixed value :-(

**Comment by ledermann** [comment]
Currently, the [Tibber-Collector](https://github.com/solectrus/tibber-collector) is only intended for use with the [SENEC-Charger](https://github.com/solectrus/senec-charger). The dynamic prices are not yet included in the cost calculation. However, this is planned, at least in the long term, see https://github.com/solectrus/solectrus/issues/2150.
